guides/common/modules/proc_uninstalling-the-foreman-templates-plugin.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Uninstalling_the_Foreman_Templates_Plugin_{context}"]
-= Uninstalling the Foreman Templates Plug-in
+= Uninstalling the Foreman templates plugin
```

Comment:
```suggestion
= Uninstalling the Foreman templates plug-in
```

Reply:
Won't do because there are too many occurrences of 'plugin'. However, IBM Style Guide says to use "plug-in", so that will be a separate PR.